### PR TITLE
feat(cli): add SNS notification arns in deploy command

### LINKED ISSUE #2528: Add notificationArn to a Stack
[This](https://docs.aws.amazon.com/cdk/latest/guide/cfn_layer.html) doesn't apply to a Stack type (`Stack cannot be cast to software.amazon.awscdk.CfnResource`). 
How can I add [notificationArns](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-stack.html#cfn-cloudformation-stack-notificationarns) to a Cdk Stack? 

I see that this can be set on a CfnStack but it isn't clear whether to convert between the two or something else. I might be missing something obvious.

### PR DESCRIPTION
**Pull request by cohalz**
It will be able to send notifications when creating/updating stacks

Fixes #2528 

----

Please read the [contribution guidelines](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md) and follow the pull-request checklist.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk/bin/cdk.ts [hunk 1]
```diff
@@ -43,6 +43,7 @@ async function parseCommandLineArguments() {
     .option('path-metadata', { type: 'boolean', desc: 'Include "aws:cdk:path" CloudFormation metadata for each resource (enabled by default)', default: true })
     .option('asset-metadata', { type: 'boolean', desc: 'Include "aws:asset:*" CloudFormation metadata for resources that user assets (enabled by default)', default: true })
     .option('role-arn', { type: 'string', alias: 'r', desc: 'ARN of Role to use when invoking CloudFormation', default: undefined, requiresArg: true })
+    .option('notification-arns', { type: 'array', alias: 'n', desc: 'ARNs of SNS topics that CloudFormation associates with the stack', nargs: 1 })
```
Comment (shivlaks) [inline packages/aws-cdk/bin/cdk.ts]: I'd prefer leaving out the alias since I don't think it's intuitive that `n` would map to notification-arns.

**Comment by shivlaks** [comment]
> I am wondering if it would be useful to be able to specify this when a stack is defined in code (not only in the CLI).

I think it would be a useful addition as it's an input to stack creation

**Comment by shivlaks** [comment]
Is this PR still being worked on?

**Comment by konstantinj** [comment]
would love to see this in the next version

**Comment by shivlaks** [comment]
@konstantinj I'm planning on picking this up if I don't get a response this week.

**Comment by shivlaks** [comment]
superseded by #4420

**Comment by pgollucci** [comment]
> I am wondering if it would be useful to be able to specify this when a stack is defined in code (not only in the CLI).

For sure. the CLI is actually useless for my use case. following up on #4420

**Pull request closed without merging**